spec/services/youtube_service_spec.rb

```diff
@@ -15,7 +15,7 @@
 
     it "playlist_items_info" do
       playlist_items_info = @service.playlist_items_info(@id)
-      expect(playlist_items_info[:items].length).to eq(13)
+      expect(playlist_items_info[:items].length).to eq(14)
```

Comment:
not sure, but this might be an example of an API updating, perhaps they added another video causing our initial call to fail

Reply:
that was my guess when I got this error earlier today

Reply:
it is now at 15 so I think they keep adding to fuck with us...or someone is adding videos as part of their app?